.circleci/config.yml

```diff
@@ -57,8 +57,8 @@ aliases:
       - <<: *install_node_modules
       - run: yarn list react
       - run: ./scripts/assert-changed-files.sh "packages/*|.circleci/*"
-      - run: yarn jest -w 1 --ci
-      - run: GATSBY_DB_NODES=loki yarn jest -w 1 --ci
+      - run: node --max-old-space-size=2048 ./node_modules/.bin/jest -w 1 --ci
```

Comment:
Did something in this PR cause the CLI to use more memory? If so -- why?

Reply:
More tests I guess. I'm not sure exactly what triggered this. I looked into it and talked with Peter a little bit. I think there are a couple issues:

1. Jest holds every test and referenced module in memory. So The more tests and code we add, the heavier the footprint will be for jest. (ew)
1. There may be memory leaks in our tests. I ran jest with `logHeapUsage` and ran with the node debugger and saw the memory grow with every test. I talked with Peter about it and he advised to just raise the memory and not to sink too much time into it.